src/platform/qt/ts/mgba-pt_BR.ts

```diff
@@ -117,27 +117,27 @@ Game Boy Advance é uma marca registrada da Nintendo Co., Ltd.</translation>
     <message>
         <location filename="../BattleChipView.ui" line="121"/>
         <source>Remove</source>
-        <translation>Excluir</translation>
+        <translation type="unfinished">Remover</translation>
     </message>
     <message>
         <location filename="../BattleChipView.ui" line="138"/>
         <source>Gate type</source>
-        <translation>Gate type</translation>
+        <translation type="unfinished">Tipo de portal</translation>
     </message>
     <message>
         <location filename="../BattleChipView.ui" line="145"/>
         <source>Ba&amp;ttleChip Gate</source>
-        <translation>Ba&amp;ttleChip Gate</translation>
+        <translation type="unfinished">Portal do Ba&amp;ttleChip</translation>
     </message>
     <message>
         <location filename="../BattleChipView.ui" line="155"/>
         <source>Progress &amp;Gate</source>
-        <translation>Progress &amp;Gate</translation>
+        <translation type="unfinished">Portal do &amp;Progresso</translation>
     </message>
     <message>
         <location filename="../BattleChipView.ui" line="162"/>
         <source>Beast &amp;Link Gate</source>
-        <translation>Beast &amp;Link Gate</translation>
```

Comment:
These are specific product names and may not make sense to translate.

Reply:
yes, after all it's between >   and   <. ;)

Reply:
I think you've misunderstood me, and you also didn't answer my other question.

Reply:
nope, i didnt change the status, i translated the file using qtlinguist, any changes about statuses made in the file was made by the program, not me.

Reply:
Well if it wasn't intentional then you should tell Qt Linguist to change them back. The program doesn't have a mind of its own.

Reply:
the problem is: qtlinguist doesnt have a web site, at least there is no link in softpedia where i download it.

Reply:
You moved several strings from finished to unfinished, and then told me it wasn't intentional, and when I asked you to flip them back, you gave not much of an explanation of why you couldn't change them back. I cannot accept it if you don't change it back.

Reply:
i'll change them but what damage this causes to mgba?

Reply:
The specific problem is that the new translation page (https://hosted.weblate.org/projects/mgba/mgba-qt/, you can use it in the future instead of a pull request) marks "unfinished" translations as "untranslated", which makes things very confusing.

Reply:
oh i see, thanks for the info, didnt know about the page.